website/pages/docs/k8s/installation/multi-cluster/overview.mdx

```diff
@@ -0,0 +1,73 @@
+---
+layout: docs
+page_title: Multi-Cluster Federation Overview
+sidebar_title: Overview
+description: >-
+  Installing on multiple Kubernetes clusters.
+---
+
+# Multi-Cluster Federation Overview
+
+In Consul, federation is the act of joining two or more Consul datacenters.
+When datacenters are joined, Consul servers in each datacenter can communicate
+with one another. This enables the following features:
+
+- Services on all clusters can make calls to each other through Consul Service Mesh.
+- [Intentions](/docs/connect/intentions) can be used to enforce cross-cluster rules about which services can communicate.
```

Comment:
This might be a little misleading - Intentions can't actually enforce which DCs are allowed to talk currently which is what this seems to imply to my reading.

Perhaps:
```suggestion
- [Intentions](/docs/connect/intentions) can be used to enforce rules about which services can communicate across all clusters.
```